l3build.dtx

```diff
@@ -1662,35 +1663,40 @@
 %
 % \begin{variable}{options}
 %   The |options| table holds the values passed to \pkg{l3build} at the
-%   command line. The possible entries in the table are given in the table
-%   below.
+%   command line. Each possible \meta{entry} given below corresponds to
+%   |--|\meta{entry} given at the command line, except |names| that
+%   corresponds to \meta{name(s)} for |check|, |doc| and |save| targets.  
 %   \begin{center}
 %   \begin{tabular}{ll}
 %     \toprule
 %     Entry & Type \\
 %     \midrule
-%       \var{config}        & Table   \\
-%       \var{date}          & String  \\
-%       \var{dirty}         & Boolean \\
-%       \var{dry-run}       & Boolean \\
-%       \var{email}         & String  \\
-%       \var{engine}        & Table   \\
-%       \var{epoch}         & String  \\
+%       \var{config}        & table   \\
+%       \var{date}          & string  \\
+%       \var{dirty}         & boolean \\
+%       \var{dry-run}       & boolean \\
+%       \var{email}         & string  \\
+%       \var{engine}        & table   \\
+%       \var{epoch}         & string  \\
 %       \var{file}          & string  \\
-%       \var{first}         & Boolean \\
-%       \var{full}          & Boolean \\
-%       \var{halt-on-error} & Boolean \\
-%       \var{help}          & Boolean \\
+%       \var{first}         & boolean \\
+%       \var{full}          & boolean \\
+%       \var{halt-on-error} & boolean \\
+%       \var{help}          & boolean \\
 %       \var{message}       & string  \\
-%       \var{names}         & Table   \\
-%       \var{quiet}         & Boolean \\
-%       \var{rerun}         & Boolean \\
-%       \var{shuffle}       & Boolean \\
-%       \var{stdengine}     & Boolean \\
-%       \var{texmfhome}     & String  \\
+%       \var{names}         & table \\
+%       \var{quiet}         & boolean \\
+%       \var{rerun}         & boolean \\
+%       \var{shuffle}       & boolean \\
+%       \var{stdengine}     & boolean \\
+%       \var{texmfhome}     & string  \\
```

Comment:
These are deliberately capitalised as the cell starts a statement

Reply:
Here the type cannot really be capitalized because it refers to 2 things
1) the output of the lua function `type`
2) section 5.6 "Each entry needs a type one of boolean, string or table.
So these are not words, these are case sensitive keywords that cannot be capitalized.
In fact they should be formatted as in section 5.6: I'll commit a change.

Reply:
Is this section 5.6 still useful?